scripts/helpers.ts

```diff
@@ -28,3 +112,73 @@ export async function getLegacyTokenHolders(
   }
   return [addresses, attestationTypes] as const
 }
+
+export async function getLegacyRegisterEntanglementCalldata(
+  address: string,
+  provider: Provider,
+  startBlock: Parameters<KetlAttestation['queryFilter']>[1] = 0,
```

Comment:
Maybe we can shortcut those types?

Reply:
I don't think we'd use them outside of this context, so I'm not sure we need that.